src/models/data.py

```diff
@@ -234,6 +247,18 @@ def ds_folder_to_np(self,
         return ModelArrays(x=x_data, y=y_np, x_vars=list(x.data_vars),
                            y_var=list(y.data_vars)[0])
 
+    @staticmethod
+    def _add_surrounding(x: xr.Dataset, surrounding_pixels: int) -> xr.Dataset:
```

Comment:
So in the `dataloader` this PR adds a new variable which is the `shifted_varname` so we have `lat1_lon1_VHI` for the historical months and `lat1_lon1_precip` for the historical months.

Does you expect it will work with the `nowcast` experiment?

Reply:
It should - `add_surrounding` is called in `ds_to_np` before the data is split into historical and current, so the current should also have the surrounding pixels.

The only thing that will need to change is that we will need to use the `endswith` check to eliminate VHI, since an exact match will miss the `lat_1_lon_1_VHI` variables.